docs/docs/debug/upgrade.md

```diff
@@ -0,0 +1,324 @@
+---
+id: upgrade
+title: Ory Kratos Upgrade Guide
```

Comment:
The user already knows that they are looking at kratos. How about: "Applying Upgrades"